app/controllers/favourites_controller.rb

```diff
@@ -0,0 +1,33 @@
+class FavouritesController < ApplicationController
+  before_action :authenticate_user
+
+  def index
+
+  end
```

Comment:
This is for listing posts that a signed in user has favourited. I will add some comments later